postcss.config.js

```diff
@@ -1,7 +1,20 @@
+const autoprefixer = require('autoprefixer');
+const assets = require('postcss-assets');
+
+const CF_DIST = process.env.CLOUDFRONT_DIST;
```

Comment:
Throwing my 2 cents in here: could always whitelabel CF with a domain we control rather that using the direct CF hostname, that way if the delivery mechanism changes we can transparently change it at the DNS level or take the time to recompile this.